CMakeLists.txt

```
@@ -104,6 +104,9 @@ if (BUILD_EXAMPLES)
     add_executable(json-schema-validate app/json-schema-validate.cpp)
     target_link_libraries(json-schema-validate json-schema-validator)
 
+    install(TARGETS json-schema-validate
+        DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Pour rester compatible.

```suggestion
        DESTINATION bin)
```